Lab Officer cannot create Cases for Event Participants

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
Lab Officer is unable to create Cases for Event Participants because the system crashes when he tries to perform the action.

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
1. Log in with an Admin,NatUser;
2. Create an Event and an Event Participant;
3. Add a Sample (pending) for the Event Participant and select the Laboratory that the Lab Officer user is associated to;
4. Log in with the Lab Officer user;
5. Navigate to the Events Directory, open an Event and select the Event Participants tab;
6. Click on the 'Create' button under the Case ID;
7. Fill in the mandatory data from the 'Create new Case' form and Save;

### Actual Behavior
An error message is displayed and the Case is not created;

### Expected Behavior
The Case should get created successfully.

### Screenshots
![image](https://user-images.githubusercontent.com/89522688/167099077-7e8877fb-4778-4c16-90f6-34d98f1a1832.png)

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: Windows 10
* SORMAS version: Sormas 1.71.0-SNAPSHOT (04faaf1)
* Android version/Browser: Chrome
* Server URL: local server
* User Role: Lab Officer

### Additional Information
- This issue was discovered when testing ticket #8820.
- Stacktrace for this issue:
[stacktrace.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/8638761/stacktrace.txt)
```
2022-05-04T14:40:32.377964200Z 	at de.symeda.sormas.api.event.__EventParticipantFacade_Remote_DynamicStub.getMatchingEventParticipants(de/symeda/sormas/api/event/__EventParticipantFacade_Remote_DynamicStub.java)
2022-05-04T14:40:32.377989800Z 	at de.symeda.sormas.api.event._EventParticipantFacade_Wrapper.getMatchingEventParticipants(de/symeda/sormas/api/event/_EventParticipantFacade_Wrapper.java)
2022-05-04T14:40:32.377999200Z 	at de.symeda.sormas.ui.caze.CaseController.convertSamePersonContactsAndEventparticipants(CaseController.java:312)
2022-05-04T14:40:32.378004400Z 	at de.symeda.sormas.ui.caze.CaseController.lambda$createFromEventParticipant$0(CaseController.java:204)
2022-05-04T14:40:32.378009200Z 	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:672)
2022-05-04T14:40:32.378013600Z 	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.doCommit(CommitDiscardWrapperComponent.java:515)
2022-05-04T14:40:32.378018500Z 	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:450)
2022-05-04T14:40:32.378024100Z 	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:546)
2022-05-04T14:40:32.378029700Z 	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:371)
...
2022-05-04T14:40:32.378943600Z Caused by: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
...
2022-05-04T14:40:32.379105800Z Caused by: org.postgresql.util.PSQLException: ERROR: column "eventparti0_.reportinguser_id" must appear in the GROUP BY clause or be used in an aggregate function
2022-05-04T14:40:32.379110500Z   Position: 293
2022-05-04T14:40:32.379114500Z 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
2022-05-04T14:40:32.379118900Z 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
2022-05-04T14:40:32.379123300Z 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
2022-05-04T14:40:32.379127500Z 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
2022-05-04T14:40:32.379131600Z 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
2022-05-04T14:40:32.379135600Z 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:166)
2022-05-04T14:40:32.379140200Z 	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:118)
2022-05-04T14:40:32.379144500Z 	at jdk.internal.reflect.GeneratedMethodAccessor283.invoke(Unknown Source)
2022-05-04T14:40:32.379148900Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-05-04T14:40:32.379153400Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-05-04T14:40:32.379157500Z 	at org.postgresql.ds.PGPooledConnection$StatementHandler.invoke(PGPooledConnection.java:441)
2022-05-04T14:40:32.379161800Z 	at com.sun.proxy.$Proxy744.executeQuery(Unknown Source)
2022-05-04T14:40:32.379165900Z 	at com.sun.gjc.spi.jdbc40.PreparedStatementWrapper40.executeQuery(PreparedStatementWrapper40.java:658)
2022-05-04T14:40:32.379170100Z 	at jdk.internal.reflect.GeneratedMethodAccessor283.invoke(Unknown Source)
2022-05-04T14:40:32.379186000Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-05-04T14:40:32.379193600Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-05-04T14:40:32.379198100Z 	at com.sun.gjc.spi.jdbc40.ProfiledConnectionWrapper40$1.invoke(ProfiledConnectionWrapper40.java:457)
2022-05-04T14:40:32.379202600Z 	at com.sun.proxy.$Proxy745.executeQuery(Unknown Source)
2022-05-04T14:40:32.379207000Z 	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.extract(ResultSetReturnImpl.java:57)
2022-05-04T14:40:32.379211300Z 	... 184 more
```

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbTest.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbTest.java
@@ -32,10 +32,6 @@ import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 
-import de.symeda.sormas.api.caze.VaccinationInfoSource;
-import de.symeda.sormas.api.common.DeletionDetails;
-import de.symeda.sormas.api.common.DeletionReason;
-import de.symeda.sormas.api.event.SimilarEventParticipantDto;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
 import org.junit.Assert;
@@ -48,12 +44,15 @@ import de.symeda.sormas.api.caze.VaccinationStatus;
 import de.symeda.sormas.api.caze.Vaccine;
 import de.symeda.sormas.api.caze.VaccineManufacturer;
 import de.symeda.sormas.api.clinicalcourse.HealthConditionsDto;
+import de.symeda.sormas.api.common.DeletionDetails;
+import de.symeda.sormas.api.common.DeletionReason;
 import de.symeda.sormas.api.event.EventDto;
 import de.symeda.sormas.api.event.EventInvestigationStatus;
 import de.symeda.sormas.api.event.EventParticipantCriteria;
 import de.symeda.sormas.api.event.EventParticipantDto;
 import de.symeda.sormas.api.event.EventParticipantExportDto;
 import de.symeda.sormas.api.event.EventStatus;
+import de.symeda.sormas.api.event.SimilarEventParticipantDto;
 import de.symeda.sormas.api.event.TypeOfPlace;
 import de.symeda.sormas.api.immunization.ImmunizationDto;
 import de.symeda.sormas.api.immunization.ImmunizationManagementStatus;
@@ -169,7 +168,7 @@ public class EventParticipantFacadeEjbTest extends AbstractBeanTest {
 	@Test
 	public void testGetMatchingEventParticipants() {
 		TestDataCreator.RDCF rdcf = creator.createRDCF();
-		UserDto user = creator.createUser(rdcf, UserRole.LAB_USER);
+		UserDto user = creator.createUser(rdcf, creator.getUserRoleReference(DefaultUserRole.LAB_USER));
 		user.setLaboratory(rdcf.facility);
 		getUserFacade().saveUser(user);
 		loginWith(user);

